src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let totalValue = 0;
+
+for (let i = 0; i < populations.length; i++) {
+  const value = populations[i].textContent.replace(/,/g, '');
+  const valueInNumber = +value;
+
+  if (typeof valueInNumber === 'number') {
```

Comment:
The condition `typeof valueInNumber === 'number'` is always true since `valueInNumber` is explicitly converted to a number. Consider using `!isNaN(valueInNumber)` to check if the conversion resulted in a valid number.